pkg/api/defaults-kubelet.go

```diff
@@ -163,6 +163,18 @@ func (cs *ContainerService) setKubeletConfig(isUpgrade bool) {
 		setMissingKubeletValues(cs.Properties.MasterProfile.KubernetesConfig, o.KubernetesConfig.KubeletConfig)
 		addDefaultFeatureGates(cs.Properties.MasterProfile.KubernetesConfig.KubeletConfig, o.OrchestratorVersion, "", "")
 
+		if isUpgrade && common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.14.0") {
```

Comment:
please add unit tests to cover the added logic

Reply:
Two more test functions are added.